test/specs/ngeo/modules/import/WmtsGetCapDirective.spec.js

```diff
@@ -1,94 +0,0 @@
-/* eslint-disable max-len */
-describe('ngeo.wmtsGetCapDirective', function() {
-  var elt, scope, parentScope, map, $rootScope, $compile, $window;
-
-  var loadDirective = function() {
-    parentScope = $rootScope.$new();
-    var tpl = '<div ngeo-wmts-get-cap="getCap" ngeo-wmts-get-cap-map="map" ngeo-wmts-get-cap-options="options"></div>';
-    elt = $compile(tpl)(parentScope);
-    $rootScope.$digest();
-    scope = elt.isolateScope();
-  };
-
-  var wmtsBasic;
-
-  before(function(done) {
-    $.get('base/test/data/wmts-basic.xml', function(response) {
-      wmtsBasic = response;
-      done();
-    });
-  });
-
-  beforeEach(function() {
-
-    module(function($provide) {
-      $provide.value('gaLayers', {});
-      $provide.value('gaTopic', {});
-      $provide.value('gaLang', {
-        get: function() {
-          return 'somelang';
-        },
-        getNoRm: function() {
-          return 'somelang';
-        }
-      });
-    });
-
-    inject(function($injector) {
-      $rootScope = $injector.get('$rootScope');
-      $compile = $injector.get('$compile');
-      $window = $injector.get('$window');
-    });
-
-    map = new ol.Map({});
-    map.setSize([600, 300]);
-  });
-
-  describe('loads a correct WMTS GetCapabilities', function() {
-
-    beforeEach(function() {
-      $rootScope.map = map;
-      $rootScope.getCap = wmtsBasic;
-      $rootScope.options = {};
-      loadDirective();
-    });
-
-    it('creates html elements', function() {
-      expect(elt.find('[ngeo-wmts-get-cap-item]').length).to.be(3);
-      expect(elt.find('.fa-sort-by-alphabet').length).to.be(1);
-      expect(elt.find('.ngeo-add').length).to.be(1);
-    });
-
-    it('has good scope values', function() {
-      expect(scope.map).to.be(map);
-      expect(scope.layers.length).to.be(3);
-      expect(scope.limitations).to.be(undefined);
-      expect(scope.userMsg).to.be(undefined);
-      expect(scope.options.layerSelected).to.be(null);
-      expect(scope.options.layerHovered).to.be(null);
-      expect(scope.addLayerSelected).to.be.a(Function);
-      expect(scope.getAbstract).to.be.a(Function);
-    });
-  });
-
-  describe('loads an invalid WMTS GetCapabilities', function() {
```

Comment:
this one is missing

Reply:
same here https://github.com/geoadmin/mf-geoadmin3/blob/5a0534eeb919d89d8c7fd54a8fd04a752f32f6cd/test/specs/import/WmtsGetCapDirective.spec.js#L74

Reply:
The test was already duplicated in master